scipp-developer-no-mantid.yml

```diff
@@ -17,7 +17,7 @@ dependencies:
   - appdirs
   - ipympl
   - ipywidgets
-  - matplotlib
+  - matplotlib=3.2.2
```

Comment:
Is it worth linking to the issue here so it is obvious why it was pinned?

Reply:
This is actually more because of JupyterLab, so i don't have an issue to link to, but I can add a comment